common/git.go

```diff
@@ -47,10 +48,19 @@ func CheckForGit(cwd string) error {
 // GetSSHRemoteURL gets the URL of the given remote in the form
 // "git@github.com:[USER]/[REPOSITORY].git"
 func GetSSHRemoteURL(url string) string {
```

Comment:
maybe this function should return an error in case anything unexpected comes up 🤔 not sure how we'd check though